contributing.md

```diff
@@ -0,0 +1,56 @@
+# Contributing
+
+We encourage any in the team (or outside of it!) to contribute to this project and make it better. So, to help we've put together the following guide.
+
+> Don't panic though! If you are still unsure just crack on with your contribution. You can't break anything hear and we'd rather see your idea than miss out because we didn't write this clearly enough.
+
+## Guides and documents
+
+Any suggested change to the project's contents, by which we mean the files and folders, should be done using a PR. Have a read of [ways of working](/ways_of_working.md) first. This is because the format for commits and the way we process PR's here is the same as for any of our code repos.
+
+Assign it a [label](https://docs.github.com/en/issues/using-labels-and-milestones-to-track-work/managing-labels) but again don't panic about correctness. Some suggestions would be
+
+- correcting a typo or incorrect fact? `bug`
```

Comment:
I would suggest this be automated: 

Using conventional commits - https://github.com/action-runner/conventional-labeler

And this action for folder / file changes - https://github.com/actions/labeler (could be useful for the cypress test changes in the UI repo)

Reply:
https://www.conventionalcommits.org/en/v1.0.0/

Reply:
TIL about conventional commits and why @jonathangoulding 's commits start with `chore:` and `feat:`! 😂 

However, (and this is IMHO), I think we have a lot going on right now so I don't want to complicate our processes further. This applies to what we do and our workflows.

It's purely subjective; I think asking for a _fuller_ commit message that isn't just a link to the JIRA ticket is doable. Asking folks to follow a specific spec might be a step too far right now 🤷 .

[Pull Request Labeler](https://github.com/actions/labeler) is definitely worth look but

- the context for this is labelling an issue not a PR. So, not sure it'd work in this context
- I want to get a handle on the CI pipeline across the reps are complete the other issues related to that before I think of then expanding functionality again

So, right now this goes on my backlog for things to revisit once we've (I've!) got more of a handle on this service.

Hope that's ok